views/components/ColorPicker.jsx

```diff
@@ -18,22 +13,49 @@ const ColorPicker = ({
 
   return (
     <div className={classes.swatchContainer}>
-      <Typography variant='caption' className={classes.colorLabel}>
-          Color
+      <Typography variant="caption" className={classes.colorLabel}>
+        Color
       </Typography>
       <Paper
         className={classes.swatch}
         style={{ backgroundColor: color }}
         onClick={() => setColorPickerToggle(true)}
       />
-    {isColorPickerOpen && (
-      <div className={classes.popover} ref={popover}>
-        <HexColorPicker 
-          color={color} 
-          onChange={(newColor) => onColorChange({target: { name: colorKey, value: newColor }}, idx)} 
-        />
-      </div>
-    )}
+      {isColorPickerOpen && (
+        <div
+          className={classes.popover}
+          ref={popover}
+          key={idx + 'color_picker'}
+        >
+          <HexColorPicker
+            color={color}
+            onChange={(newColor) =>
+              onColorChange(
+                { target: { name: colorKey, value: newColor } },
+                idx
+              )
```

Comment:
I think it's worth changing the `onColorChange` prop function to work with the color picker instead of having to make a fake event from the new color to make it work with the previous implementation.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/183/commits/575b52832ef21a014787ebbec9f099dcbd288fbd

Reply:
Not sure why but now that I'm using radios instead of buttons the warnings are gone

Reply:
https://github.com/AMP-SCZ/dpdash/pull/183/commits/79ef2008456c1be80c4e790477d0d387f1ebff1b